pilot/pkg/networking/core/v1alpha3/accesslog.go

```diff
@@ -41,6 +42,14 @@ const (
 		"\"%REQ(USER-AGENT)%\" \"%REQ(X-REQUEST-ID)%\" \"%REQ(:AUTHORITY)%\" \"%UPSTREAM_HOST%\" " +
 		"%UPSTREAM_CLUSTER% %UPSTREAM_LOCAL_ADDRESS% %DOWNSTREAM_LOCAL_ADDRESS% " +
 		"%DOWNSTREAM_REMOTE_ADDRESS% %REQUESTED_SERVER_NAME% %ROUTE_NAME%\n"
+	// EnvoyTextLogFormatIstio18 format for envoy text based access logs for Istio 1.8 onwards
+	EnvoyTextLogFormatIstio18 = "[%START_TIME%] \"%REQ(:METHOD)% %REQ(X-ENVOY-ORIGINAL-PATH?:PATH)% " +
```

Comment:
will this break people parsing output, or are they expected to be using json format?

Reply:
that's possible but I don't think we treat the log format as a stable API otherwise it seems we can never really update it? Also I think the user is probably using their customized access log format if they're going to parse it?

Reply:
I think I agree with you, just wanted to put out the comment in case someone else disagrees. I have used https://github.com/nitishm/engarde which does this but that doesn't mean its a valid use case.